build.rs

```diff
@@ -0,0 +1,15 @@
+#[cfg(not(any(target_os = "macos", windows)))]
+extern crate pkg_config;
+
+#[cfg(not(any(target_os = "macos", windows)))]
+fn main() {
+    let library_version = pkg_config::probe_library("freetype2").unwrap().version;
```

Comment:
The unwrap here scares me. What does it do? I don't think it's the best idea.

Also can't we just use the following?

```rust
let enough = pkg_config::Config::new().atleast_version("2.8.0").probe("freetype2").is_ok();
```

While the `>=` vs `atleast_version` might not make a big difference, being able to specify the actual freetype version would.

Reply:
Well, we can do something like that, I guess.

Reply:
If that actually works, it would be much better I think.

Reply:
I've updated and it works 'fine'.